### PR TITLE
Splash Screen with NavController

### PR DESCRIPTION
**Pull request by ted3x**
Created package UI with packages inside: movies_dashboard, splash_screen.
Created Main Activity.
Created SharedViewModel for activity and fragments.
Created NavController for navigation.

### CODE REVIEW

File: app/src/main/java/com/c0d3in3/movieapp/ui/SharedViewModel.kt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package com.c0d3in3.movieapp.ui
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import java.util.logging.Handler
+
+class SharedViewModel : ViewModel() {
+
+    val isDataLoaded: MutableLiveData<Boolean> = MutableLiveData()
```
Comment (Edna001) [inline app/src/main/java/com/c0d3in3/movieapp/ui/SharedViewModel.kt:9]: val isDataLoaded = MutableLiveData< Boolean >() აქ პირდაპირ ასე შეგიძლია დაწერო

File: app/src/main/java/com/c0d3in3/movieapp/ui/MainActivity.kt [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProvider
+import androidx.navigation.NavController
+import androidx.navigation.Navigation
+import com.c0d3in3.movieapp.R
+import kotlinx.android.synthetic.main.activity_main.*
+
+class MainActivity : AppCompatActivity() {
+
+    private lateinit var sharedViewModel: SharedViewModel
```
Comment (Edna001) [inline app/src/main/java/com/c0d3in3/movieapp/ui/MainActivity.kt:15]: არ მომწონს ეს სახელი :D

**Review by Edna001 — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/c0d3in3/movieapp/ui/MainActivity.kt (hunk 2 above)
Comment (ted3x) [reply app/src/main/java/com/c0d3in3/movieapp/ui/MainActivity.kt:15]: Yes, I will change it.

File: app/src/main/java/com/c0d3in3/movieapp/ui/SharedViewModel.kt (hunk 1 above)
Comment (ted3x) [reply app/src/main/java/com/c0d3in3/movieapp/ui/SharedViewModel.kt:9]: It's unnecessary to explicit type of variable in this situation?

Comment (Edna001) [reply app/src/main/java/com/c0d3in3/movieapp/ui/SharedViewModel.kt:9]: no, it's unnecessary

**Review by anarusadzeSpace — APPROVED** [review]

File: app/src/main/java/com/c0d3in3/movieapp/ui/SharedViewModel.kt (hunk 1 above)
Comment (ted3x) [reply app/src/main/java/com/c0d3in3/movieapp/ui/SharedViewModel.kt]: Changed

File: app/src/main/java/com/c0d3in3/movieapp/ui/MainActivity.kt (hunk 2 above)
Comment (ted3x) [reply app/src/main/java/com/c0d3in3/movieapp/ui/MainActivity.kt]: Changed

**Pull request merged**